src/components/EpisodeList.tsx

```diff
@@ -0,0 +1,39 @@
+import IEpisode from "../utils/i-episode";
+import { episodeCode } from "../utils/episode-code";
+import { omitTags } from "../utils/omit-tags";
+import { useState } from "react";
+
+interface EpisodeListProps {
```

Comment:
Could we not use IEpisode directly instead of defining a new interface over here?

Reply:
Defining the new interface makes passing the props to this component easier. Otherwise we need to pass in id, summary, etc.